grouper/fe/handlers/audits_complete.py

```diff
@@ -31,37 +31,37 @@ def post(self, audit_id):
             if argument.startswith("audit_"):
                 edges[int(argument.split("_")[1])] = self.request.arguments[argument][0].decode()
 
-        for member in audit.my_members():
-            if member.id in edges:
+        for ami in get_group_audit_members_infos(self.session, audit.group):
```

Comment:
In the spirit of the new architecture and its extreme verbosity, personal preference for `ami` -> `audit_member_info` :)

Reply:
hah